### PR TITLE
style(app): add new button variations for the app

### PR DESCRIPTION
**Pull request by smb2268**
# Overview
Add new styled variations of Btn to use in the app

# Changelog
1. Added new button variations and stories
2. Updated all instances of PrimaryBtn and SecondaryBtn in the Protocol and Run tabs to use these versions

# Review requests

- [ ] Run the component storybook and check that the designs match the Figma linked in the ticket
- [ ] Run through a protocol flow to make sure there are no weird styling issues

# Risk assessment

Low

### CODE REVIEW

**Review by emilywools — APPROVED** [review]
this looks good to me!!! thank you Sarah this looks so much better!!

File: components/src/primitives/Btn.tsx [hunk 1]
```diff
@@ -115,6 +115,111 @@ export const SecondaryBtn: BtnComponent = styled(Btn)`
   ${styleProps}
 `
 
+/**
+ * New primary button variant used in app
+ *
+ * @component
+ */
```
Comment (shlokamin) [inline components/src/primitives/Btn.tsx:122]: if this is specifically meant for the app (as the name + description suggest) i think this should be moved into the app directory

Comment (shlokamin) [reply components/src/primitives/Btn.tsx:122]: i imagine we'll eventually want these buttons to be the defaults used in our other products too (we'll certainly be using these for next gen app stuff), maybe it's just worth a name change...

Comment (smb2268) [reply components/src/primitives/Btn.tsx:122]: Yeah I thought about putting it in the app directory but wanted us to be able to access the styles in the components library storybook. I could rename them to "NewPrimaryBtn" which is what Brian initially suggested in standup this morning?

**Pull request merged**